<html><head>
<style> table, th, td {
  border: 1px solid black;
  border-collapse: collapse;
}th {
  background: lightblue;
}</style>
</head> <body><h1>SATD</h1><table><tr><th>satd id</th> <th>satd instance id</th>  <th>project</th> <th>committer name </th> <th> Commit Hash</th> <th>old comment</th> <th>New Comment</th> <th>resolution</th> <th>Method Signature</th> <th>Method Declaration</th> <th>Method Body</th> </tr><tr><td>1949</td> <td>-1422651031</td><td>apache/camel</td><td>Willem Ning Jiang</td><td>613d4b7eca55458b46009f45e2ba5a78cc6289e5</td> <td>TODO Following assertion should be true while it is not. Instead
exception handler set in BatchProcessor is used and it logs
by default.</td> <td>second aggregated</td> <td>SATD_REMOVED</td> <td>testExceptionInFlow(String)</td> <td>private void testExceptionInFlow(String startUri) throws Exception</td> <td>
    // failed first aggregation
    getMockEndpoint("mock:handled").expectedMessageCount(1);
    // second aggregated
    getMockEndpoint("mock:result").expectedBodiesReceived("Hello World");
    template.sendBodyAndHeader(startUri, "Damn", "id", 1);
    template.sendBodyAndHeader(startUri, "Hello World", "id", 1);
    assertMockEndpointsSatisfied();
</td> </tr></table></body></html>